fiat-roles/src/test/groovy/com/netflix/spinnaker/fiat/permissions/RedisPermissionsRepositorySpec.groovy

```diff
@@ -230,34 +230,34 @@ class RedisPermissionsRepositorySpec extends Specification {
                  .setRoles([role1] as Set))
 
     then:
-    jedis.smembers("unittests:users") == ["testUser"] as Set
-    jedis.smembers("unittests:roles:role1") == ["testUser"] as Set
+    jedis.smembers("unittests:users") == ["testuser"] as Set
```

Comment:
slight concern over what this might mean for existing data in redis but I think our fallback behaviors will handle it -- anonymous is already lower case so someone that was indexed in mixed case will trip our fallback to re-resolve their permission set

Reply:
Good callout.  Yeah I think the `PermissionsResolver` will adequately handle the fallback scenario.  Will keep an eye on this as it rolls out though.